src/classes/test_method_objects.py

```diff
@@ -36,13 +36,20 @@ def test_method_objects():
 
     # Usually, a method is called right after it is bound
     counter = MyCounter()
+    MyCounter.counter = 20
```

Comment:
can't under stand this. Are you assinging a value to class method?

Reply:
i was trying to show that variables declared outside the constructor are shared by all instances and if modified like this it will also change the values of all instances as well